docs/ReleaseProcess.md

```diff
@@ -1,21 +1,27 @@
 # Release Process
 
-This file details the steps for creating a versioned release of
-eBPF for Windows, and how to service it later.
+This document outlines the steps to create, publish and service a release of eBPF for Windows.
 
->**Note**: Currently releases are *not* production signed and therefore considered as "pre-releases". Their versioning is currently fixed to *Major Version (i.e., "X")* set to "`0`", therefore versioned as "`0.Y.Z`".
+The eBPF project follows [Semantic Versioning 2.0](https://semver.org) for versioning releases in the format `X.Y.Z`, where "`X`" and "`Y`" are the major and
+minor versions respectively. "`Z`" is the patch version for servicing releases. A release is usually made once every month. Pre-release binaries are test signed.
+Official releases will be production signed using Microsoft certificates.
+
+>**Note**: Currently the *major version* for all releases is set to "`0`" (i.e. "`0.Y.Z`") until the first official release is published with version `1.0.0`.
 
 ## Creating a new release
 
-1. Create a working branch from "`main`" on your private repo fork, and check it out (e.g., `<user>/release-X-Y-Z`).
-1. Run the following script from the root directory of the repository, within a "*Developer Poweshell for VS 2022"* instance. Make sure to follow [Semantic Versioning 2.0](https://semver.org) ("`X.Y.Z`"):
+An issue with the title `Scheduled eBPF release is due` is created on the first day of every month requesting for a new release with the minor version incremented
+every time. When this issue is triaged, a decision must be taken by the maintainers on whether to go ahead with the new monthly release. If the decision is to
+create a new release the release manager must proceed with the following process.
+1. Create a topic branch from "`main`" on your private repo fork, and check it out (e.g., `<user>/release-build`).
```

Comment:
```suggestion
1. Create a topic branch from "`main`" on your private repo fork, and check it out (e.g., `<user>/release-X-Y-Z`).
```
I'd suggest suggesting the name as before, as infusing precision especially for who does the process the first time is better.